src/Integration/WPCOMProxy.php

```diff
@@ -169,26 +194,45 @@ function ( $response, $handler, $request ) {
 					return $response;
 				}
 
-				$data = $response->get_data();
+				if ( $request->get_route() === '/wc/v3/settings/' . self::SETTINGS_GROUP ) {
+
+					$data = $response->get_data();
 
-				if ( $request->get_route() === '/wc/v3/settings/general' ) {
 					$data[] = [
-						'id'    => 'gla_target_audience',
-						'label' => 'Google for WooCommerce: Target Audience',
-						'value' => $this->options->get( OptionsInterface::TARGET_AUDIENCE, [] ),
+						'id'    => 'gla_google_connected',
+						'label' => 'Google for WooCommerce: Is Google account connected?',
+						'value' => rest_sanitize_boolean( $this->options->get( OptionsInterface::GOOGLE_CONNECTED, false ) ),
 					];
-
 					$data[] = [
-						'id'    => 'gla_shipping_times',
-						'label' => 'Google for WooCommerce: Shipping Times',
-						'value' => $this->shipping_time_query->get_all_shipping_times(),
+						'id'    => 'gla_jetpack_connected',
+						'label' => 'Google for WooCommerce: Is Jetpack connected?',
+						'value' => rest_sanitize_boolean( $this->options->get( OptionsInterface::JETPACK_CONNECTED, false ) ),
```

Comment:
Was there a specific mention / request for this to be passed?

As far as I understand if it's not connected, then data can't be pulled from the API through WPCOM. So this value will always be true.

Reply:
There was no request. 
> I added gla_jetpack_connected and gla_google_connected entries. They were not mentioned in the shared Google doc, but I believe that it could be useful to Google to check if a merchant disconnected their account. Plus, those could be considered onboarding inputs.

I thought checking if Google account is connected could be useful to Google. And running the request to a local endpoint could be useful for debugging/introspection to all GLA settings aggregated.

But maybe, as per YAGNI, I'll remove them.

Reply:
Do you think we should keep `gla_google_connected` or remove it as well?

Reply:
I think it's fine to leave it, not sure how it will be used, but the extra resource usage is insignificant if we leave it in. Unless you want to be strict on the YAGNI principle.